### ISSUE TITLE
[Support Page] Accordions in common questions section need more descriptive text and aria-expanded property

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#6360]
## Area or Component

Static site -> Support page -> accordions under the common questions section.

## Quick Summary

Accordion button is missing the aria-expanded property with the correct state (collapse|expanded) and the `more|less` text should be more descriptive. Example: "see more account management questions" |"see less account management questions".

### How to Replicate
Go to the page and run the interactive axe guided test on the accordion buttons. You can also check the DOM.

## Accessibility Violation

- [The element has missing or incorrect states or properties that are necessary for screen reader users to interact with or understand the content conveyed by the element.](chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/panel.html#)
- [The element's accessible name is missing or is not appropriate for the element's purpose.](chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/panel.html#)

## Action Requested

Add the aria-expanded attributes and the right states. Modify the aria label of the button to be more descriptive.

## Screenshots (optional)
<img width="1020" alt="Screenshot 2023-08-18 at 12 52 42 PM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/bb3eb30c-da62-43ad-969e-25519cb56e18">
